firebase-firestore/src/main/java/com/google/firebase/firestore/local/TargetData.java

```diff
@@ -96,7 +105,21 @@ public TargetData withResumeToken(ByteString resumeToken, SnapshotVersion snapsh
         purpose,
         snapshotVersion,
         lastLimboFreeSnapshotVersion,
-        resumeToken);
+        resumeToken,
+        expectedCount);
```

Comment:
`withResumeToken(...)` should specify a null `expectedCount` in the newly-created `TargetData` object.

https://github.com/firebase/firebase-js-sdk/blob/dd6683583b7c6b07586c2551f4d362b48fb83f7c/packages/firestore/src/local/target_data.ts#L108